components/Text/Body.tsx

```diff
@@ -4,7 +4,7 @@ import styled, { css } from 'styled-components';
 export type BodyProps = React.HTMLAttributes<HTMLParagraphElement> & {
 	size?: 'xsmall' | 'small' | 'medium' | 'large';
 	weight?: 'regular' | 'bold' | 'black';
-	color?: 'primary' | 'secondary' | 'tertiary' | 'positive' | 'negative' | 'preview';
+	color?: 'primary' | 'secondary' | 'tertiary' | 'quaternary' | 'positive' | 'negative' | 'preview';
```

Comment:
I don't think we need a new color type for this as it's very specific, we should probably just override the body component inside the text toggle.

Reply:
Thanks. Updated.